tests/test.py

```diff
@@ -844,11 +859,14 @@ class AsyncpgExecutor(QueryExecutor):
     BinaryFormat = 'binary'
 
     def _connect(self, loop):
+        ssl_context = ssl.create_default_context(cafile=self.connection_args.ssl_ca)
+        ssl_context.load_cert_chain(self.connection_args.ssl_cert, self.connection_args.ssl_key)
+        ssl_context.check_hostname = False
```

Comment:
it doesn't work if we will check hostname? all our certs generated for `localhost` and we connect to `localhost` in tests. did you try?

Reply:
I didn't try, but re-test all with  `check_hostname=True` and its working well.